### PR TITLE
Feature/mailchimp template Gerard Celma

### PR DESCRIPTION
**Pull request by gcelma**

### CODE REVIEW

File: staff/gerard-celma/mailchimp-template/index.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            <img src="img/main_logo.png" alt="mailchimp logo" class="header__logo">
+            <button class="button">Sign Up Free</button>
+                <nav class="header__links">
+                    <a href="#">Pricing</a>
+                    <a href="#">Features</a>
+                    <a href="#">Resources</a>
+                    <a href="#">Support</a>
+                    <a href="#">Blog</a>
+                    <div class="header__search">
+                        <p>Search</p>
```
Comment (lexbarba) [inline staff/gerard-celma/mailchimp-template/index.html:25]: Should this be an input to make a search?

File: staff/gerard-celma/mailchimp-template/index.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                </nav>
+            </div>
+            </div>
+        </div>
+    </header>
+    <main class="main">
+        <aside class="asidetop">
+            <img src="img/easy-email-newsletters-banner-png8.png" alt="momkey" class="asidetop__monkey">
+            <div class="asidetop__signup">
+                <h2>Easy Email</h2>
```
Comment (lexbarba) [inline staff/gerard-celma/mailchimp-template/index.html:37]: There was an image in the files to put that title!

File: staff/gerard-celma/mailchimp-template/style.sass [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+// Color Palette
+$blue: #46BDD5
+$light-grey: #F0F0F0
+$mid-grey: #CCD0CF
+$mid-dark-grey: #3F3F37
+$dark-grey: #181a1c
+$orange: #FF664D
+$light-orange: #ff8668
+
+// Mixins
```
Comment (lexbarba) [inline staff/gerard-celma/mailchimp-template/style.sass:12]: Mixins are used to reuse code if value changes. If it does not change, extend/inheritance should be used.

File: staff/gerard-celma/mailchimp-template/index.html (hunk 1 above)
Comment (gcelma) [reply staff/gerard-celma/mailchimp-template/index.html:25]: Truth that, did it like this to go faster ;)

File: staff/gerard-celma/mailchimp-template/index.html (hunk 2 above)
Comment (gcelma) [reply staff/gerard-celma/mailchimp-template/index.html:37]: Hostia i didn’t saw it, will check

File: staff/gerard-celma/mailchimp-template/style.sass (hunk 3 above)
Comment (gcelma) [reply staff/gerard-celma/mailchimp-template/style.sass:12]: Very good one, didn’t knew this, i’ll make the change, thank you Alexander Beard 👏🏼👏🏼👏🏼

**Pull request merged**